src/goose/console.clj

```diff
@@ -3,8 +3,14 @@
             [ring.middleware.keyword-params :as ring-keyword-params]
             [ring.middleware.params :as ring-params]))
 
+(defn- wrap-prefix-route [handler]
+  (fn [{{:keys [route-prefix]} :console-opts
+        :as req}]
+    (let [prefix-route-fn (partial str route-prefix)]
+      (handler (assoc req :prefix-route prefix-route-fn)))))
 
 (defn app-handler [{:keys [broker] :as console-opts} req]
   ((-> (partial b/handler broker)
+       wrap-prefix-route
```

Comment:
suggestion: can we add a comment here explaining why we use this middleware? It might not be clear from just looking at the definition of `wrap-prefix-route`